notebooks/2_read_write.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<p>I think it's worth mentioning explicitly that there are no parentheses after <code>dtypes</code> because it's an attribute and not a method.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Good point, added this as a note to the dataframe.</p>